content/learn/fundamental-wellbeing.md

```diff
@@ -1,73 +1,96 @@
+Fundamental Wellbeing is therefore the experience people have when they transition to this heightened human experience.
 
-[Lifeitself Podcast](https://lifeitself.org/podcast/a-scientific-approach-to-awakening-and-fundamental-wellbeing-podcast)
+## What are the criteria for experiencing Fundamental Wellbeing?
 
-[YouTube](https://youtu.be/EmsLdIH8nzU)
+Through Dr.Martins extensive interviews he was able to identify some key criteria that came with the transition to Fundamental Wellbeing that was reported across multiple faiths and religions:
 
-[Spotify](https://spotifyanchor-web.app.link/e/atCvzSYLazb)
+- A reduction in thoughts, particularly self-reflective thoughts.
+- Negative emotions would go away much faster (or negative motions do not exist at all)
+- Living in the past and future is less common, mostly living in the present
+- Feelings of persistent contentment
+- An increased sense of connectedness
 
-Part 2: Dr Jeffery Martin on A Scientific Approach to Awakening and Fundamental Wellbeing 
+## What is Dr. Martin's research? 
 
-[Lifeitself Podcast](https://lifeitself.org/podcast/a-scientific-approach-to-awakening-and-fundamental-wellbeing-part-2-jeffery-martin)
+In 2006, Dr. Martin started his PhD in Transformative Studies at the California Institute of Integral Studies determined to figure out how to become the happiest person on the planet. His approach began by surveying happy people. Not just any happy person off the street, but spiritually developed individuals from different religions and traditions which appeared to have attained full spiritual awakening. The surveys were to determine if there were psychological differences between these people and other people, however these initial findings did not indicate anything specifically unique.  
 
-[YouTube](https://youtu.be/0l_jrk0HTVY)
+Following this his team conducted live interviews with the selection of people using cognitive science techniques which enabled them to capture consistent data between different individuals across different religions, traditions and experiences that could be compared. This lead to the range of criteria of Fundamental Wellbeing, which has been continually researched by Dr. Martin. You can learn more about this in episode one of our interviews.
 
-[Spotify](https://spotifyanchor-web.app.link/e/c34gB9Bb8yb)
+## How do you reach Fundamental Wellbeing?
 
-Part 3: Dr Jeffery Martin on A Scientific Approach to Awakening and Fundamental Wellbeing 
+Those who have transitioned to Fundamental Wellbeing have done so through many gateways and environments, from deep spiritual devotion to psychedelic experiences. However, through his research Dr. Martin has developed a protocol for people to follow to transition to Fundamental Wellbeing.
 
-[Lifeitself Podcast](https://lifeitself.org/podcast/a-scientific-approach-to-awakening-and-fundamental-wellbeing-part-3-jeffery-martin)
+Learn more about is this Part 2 of the interviews.
+## What is the Martin Matrix? 
```

Comment:
Would be nice to have a picture here.

Reply:
I've included an image describing each of the locations, the the video thumbnail shows the layers. I cannot find an actual graphic of the map in its entirety.